src/main/java/com/powsybl/openloadflow/ac/AcTargetVector.java

```diff
@@ -23,9 +23,7 @@ public class AcTargetVector extends TargetVector<AcVariableType, AcEquationType>
 
     private static double getBusTargetV(LfBus bus) {
         Objects.requireNonNull(bus);
-        double targetV = bus.getHighestPriorityMainVoltageControl()
-                .map(Control::getTargetValue)
-                .orElseThrow(() -> new IllegalStateException("No active voltage control has been found for bus '" + bus.getId() + "'"));
+        double targetV = bus.getHighestPriorityTargetV().orElseThrow(() -> new IllegalStateException("No active voltage control has been found for bus '" + bus.getId() + "'"));
```

Comment:
`bus.getHighestPriorityMainVoltageControl()` method must be kept because in that method the enable and disable status is used and I don't see that in the new method. It seems that your functional need is not fully compatible with the main behavior, so mutualisation is not a good idea here. Tests are not failing and it is strange indeed...

Reply:
bus.getHighestPriorityMainVoltageControl() was only used to recover the bus target voltage, that is why I have made this modification. I do not think that the enable/disable status is needed since we have stated that in case of a contingency on an element controling a bus voltage, then the element in contingency still applies its target on this bus  voltage if another element with a lower priority also controls voltage at this very bus.

Reply:
But you decide that for your need, not for every need, the AC target vector is used by everybody. Maybe I was not clear enough but we want to keep the behavior of main branch.

Reply:
The disabled criteria on voltage control that seems to have been deleted.

Reply:
I have rolled back to previous behavior. The disabled criteria is now taken into account as in main branch.

Reply:
@annetill, following https://github.com/powsybl/powsybl-open-loadflow/pull/955#discussion_r1498806212 and @Hadrien-Godard rollback to previous behavior I am resolving the thread.

We can continue discussion in https://github.com/powsybl/powsybl-open-loadflow/pull/955#discussion_r1498806212  if needed